Protocol.md

```diff
@@ -312,6 +314,7 @@ The Error Data is typically an Exception message, but could include stringified
 | __UNSUPPORTED_SETUP__          | 0x00000002 | Some (or all) of the parameters specified by the client are unsupported by the server. Stream ID MUST be 0. |
 | __REJECTED_SETUP__             | 0x00000003 | The server rejected the setup, it can specify the reason in the payload. Stream ID MUST be 0. |
 | __CONNECTION_ERROR__           | 0x00000101 | The connection is being terminated. Stream ID MUST be 0. |
+| __CONNECTION_ERROR_NO_RETRY__  | 0x00000102 | The connection is being terminated. May __NOT__ be resumed. Stream ID MUST be 0. |
```

Comment:
I think we should only add specific errors if they will be easy for an implementation to differentiate and add logic for. Both of those errors are good, I think. Let me see about cleaning them up a little.